server/errors.go

```diff
@@ -60,23 +60,26 @@ func (e *AuthenticationError) Error() string {
 	return e.errString
 }
 
+// NoBodyError error meaning that client didn't provide body when it's required
+type NoBodyError struct{}
+
+func (*NoBodyError) Error() string {
+	return "client didn't provide request body"
+}
+
 // handleServerError handles separate server errors and sends appropriate responses
 func handleServerError(writer http.ResponseWriter, err error) {
 	log.Error().Err(err).Msg("handleServerError()")
 
 	var respErr error
 
 	switch err := err.(type) {
-	case *RouterMissingParamError:
-		respErr = responses.SendError(writer, err.Error())
-	case *RouterParsingError:
-		respErr = responses.SendError(writer, err.Error())
+	case *RouterMissingParamError, *RouterParsingError, *json.SyntaxError, *NoBodyError:
+		respErr = responses.Send(http.StatusBadRequest, writer, err.Error())
```

Comment:
Can you pls use the `SendError()` so that its unified? `Send()` is currently only used here and I will rename it in `utils` next sprint to `SendBadRequest()`

Reply:
The reason why I didn't use SendError() is that it's not really clear from the name of the function what kind of error it's gonna send. 

> I will rename it in utils next sprint to SendBadRequest()

Okay, thanks